site/404.html

```diff
@@ -0,0 +1,38 @@
+---
+layout: error
+permalink: /404.html
+sitemap: false
+---
+
+<section class="intro">
+  <div class="grid">
+    <div class="unit whole align-center">
+      <p class="first">404 - Not Found</p>
+    </div>
+  </div>
+</section>
+
+<section class="error">
+  <div class="grid">
+    <div class="unit whole center-on-mobile">
+      <p>The resource you requested was not found. Here are some links to help you find your way:</p>
+      <ul>
+        <li>
+          <a href="/">Home</a>
+        </li>
+        <li>
+          <a href="/docs/home/">Docs</a>
```

Comment:
I'd call this `Documentation` – it's `Docs` in the header because it's too wide 😄